packages/strapi-parts/src/Dialog/Dialog.stories.mdx

```diff
@@ -36,30 +36,13 @@ import AlertWarningIcon from '@strapi/icons/AlertWarningIcon';
 
 ## Usage
 
-Description...
-
 <Canvas>
   <Story name="base">
     {() => {
       const [isVisible, setIsVisible] = useState(false);
       return (
         <>
           <Button onClick={() => setIsVisible(true)}>Click me</Button>
-          <Box padding={10} background="neutral0">
-            test overflow
-          </Box>
-          <Box padding={10} background="neutral0">
-            test overflow
-          </Box>
-          <Box padding={10} background="neutral0">
-            test overflow
-          </Box>
-          <Box padding={10} background="neutral0">
-            test overflow
-          </Box>
-          <Box padding={10} background="neutral0">
-            test overflow
-          </Box>
           <Dialog onClose={() => setIsVisible(false)} title="Confirmation" isOpen={isVisible}>
```

Comment:
With the addition of the props, the overflow is taken care of so i removed the additional padding.